### PR TITLE
Add support for spaces in adb path

### PR DESCRIPTION
**Pull request by thecodrr**
My `adb` executable is located inside "Program Files (x86)" and `logkitty` fails to get it. Add quotations around the adb path seems to fix it.

### CODE REVIEW

**Comment by thecodrr** [comment]
@zamotany ping

**Comment by zamotany** [comment]
Hey @theweavrs
Thank you for the PR. Can you disable your Prettier config overrides? It's replacing the ' with ". If you tun yarn lint it will fail because of that.

**Comment by thecodrr** [comment]
@zamotany done

File: src/android/adb.ts [hunk 1]
```diff
@@ -51,7 +51,9 @@ export function getApplicationPid(
 ): number {
   let output: Buffer | undefined;
   try {
-    output = execSync(`${getAbdPath(adbPath)} shell pidof -s ${applicationId}`);
+    output = execSync(
+      `"${getAdbPath(adbPath)}" shell pidof -s ${applicationId}`
```
Comment (zamotany) [inline src/android/adb.ts]: Shouldn't `'` be used here instead of `"`?

**Review by zamotany — CHANGES_REQUESTED** [review]

File: src/android/adb.ts (hunk 1 above)
Comment (thecodrr) [reply src/android/adb.ts]: Resolved

**Review by zamotany — APPROVED** [review]

**Pull request merged**